build.sbt

```diff
@@ -117,7 +117,6 @@ lazy val core = libraryCrossProject("core")
       catsCore.value,
       catsEffectStd.value,
       catsParse.value.exclude("org.typelevel", "cats-core_2.13"),
-      crypto.value,
```

Comment:
Oh, do we not need it in core anymore?

Reply:
WebSocketHandshake was the only use in core.  We could move the improved blaze copy back to core, but nobody else uses it.

Reply:
Ah, I remember that change but didn't realize the implications. 👍

Reply:
Neither did I until the plugin caught it. Slimming down unused dependencies is a great thing, even though in this case it's reintroduced in both client and server.